cmd/tests/cinder_csi_e2e_suite_test.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
any special reason for adding this inside cmd?
could we have pkg structure like this
/test/e2e/..
/test/e2e/cmd

Reply:
The structure is a per in documentation https://kubernetes-csi.github.io/docs/functional-testing.html